### DISCUSSION TITLE
Unable to upgrade to quasar 1.19.4

### DISCUSSION THREAD

**Discussion by talkonadmin** [quasarframework/quasar#14022] [CLI - PWA mode]
While trying to update quasar to version 1.19.4 I get the error:
sync-fetch@0.4.1: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.12"

I am using node v12 since this the only version supported by quasar cli 1

**Comment by MilosPaunovic** [comment]
Upgrade `@quasar/cli` to the latest version first.

**Reply by talkonadmin** [reply]
did not help, same error (quasar/cli 1.3.2)

**Reply by MilosPaunovic** [reply]
> I am using node v12 since this the only version supported by quasar cli 1

Up the `Node` version as well, to, at least, 14.

**Reply by talkonadmin** [reply]
See this disclaimer: https://v1.quasar.dev/quasar-cli/installation
upgrade to node 14 will cause build failure, only upgrade to quasar 2 will help I guess but this is currently not an option

**Comment by MilosPaunovic** [comment]
https://github.com/quasarframework/quasar/issues/14112#issuecomment-1201620897
